src/test/CMakeLists.txt

```diff
@@ -53,6 +53,10 @@ add_subdirectory(osdc)
 add_subdirectory(pybind)
 if(${WITH_RADOSGW})
   add_subdirectory(rgw)
+  set(rgw_libs rgw_a)
+  if(WITH_RADOSGW_AMQP_ENDPOINT)
+    list(APPEND rgw_libs amqp_mock)
```

Comment:
`${rgw_libs}` is used in test/rgw/CMakeLists.txt, but this variable is set after `add_subdirectory(rgw)`

Reply:
fixed and repushed.